lib/components/parallax_component.dart

```diff
@@ -50,8 +50,9 @@ class ParallaxLayer {
   }
 
   bool loaded() => _image != null;
+  bool sized = false;
```

Comment:
This whole component will be refactored to match the Standard for asset loading that we are implementing on other features, so this will probably not be necessary in the future.